src/liberty/starterProject.ts

```diff
@@ -0,0 +1,403 @@
+import { QuickPickItem, window, Disposable, QuickInputButton, QuickInput, ExtensionContext, QuickInputButtons } from 'vscode';
+import * as devCommands from "./devCommands";
+import { getProjectOptions } from '../definitions/starterOptions';
+import * as vscode from "vscode";
+import * as fs from "fs";
+
+export async function starterProject(context: ExtensionContext) {
```

Comment:
Indentation looks off in this file, ensure you are using 4 spaces rather than tabs.
Suggestion to add a comment to the class explaining the purpose

Reply:
Fixed in https://github.com/OpenLiberty/open-liberty-tools-vscode/commit/fb3698a9f20eaaf7c4679fd2b909ed54e32b0c0a. Comments on the way..